staging/src/k8s.io/apiserver/pkg/util/webhook/webhook_test.go

```diff
@@ -722,6 +721,8 @@ func newKubeConfigFile(config v1.Config) (string, error) {
 		return "", fmt.Errorf("unable to write the Kubernetes client configuration to disk: %v", err)
 	}
 
+	configFile.Close()
```

Comment:
why not do defer here?  Will this leak if it returns from line 722?

Reply:
and no delete? I would expect this to not do a Close, but instead to call `utiltesting.RemoveTestFile(t, configFile)`